packages/lbvs_consent/lbvs_consent.2.1.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/consent"
+bug-reports: "https://github.com/UnixJunkie/consent/issues"
+dev-repo: "git+https://github.com/UnixJunkie/consent.git"
+license: "GPL"
+build: [
+  [make "-C" "src"]
+  ["jbuilder" "build" "-p" name "-j" jobs]
```

Comment:
This package depends on dune but uses jbuilder. Could it use the `dune` executable instead?

jbuilder is being slowly deprecated, see: https://dune.build/blog/second-step-deprecation/